cmd/kubectl-gadget/utils/trace.go

```diff
@@ -68,6 +68,16 @@ type TraceConfig struct {
 	// But other gadgets, like dns, can contain output only in Started state.
 	TraceOutputState string
 
+	// TraceOutput is either the name of the file when TraceOutputMode is File or
+	// the name of the external resource when TraceOutputMode is ExternalResource.
+	// Otherwise, its value is ignored.
+	TraceOutput string
+
+	// TraceInitialState is the state in which the trace should be after its
+	// creation.
+	// This field is not used by *-collector gadget but needed by others.
+	TraceInitialState string
+
```

Comment:
It's not used anywhere. What's this for?

Reply:
Hum...
It is used in another commit which belongs to #399, it seems I got confused with rebasing thing.
I will remove it, good catch!